app/grandchallenge/reader_studies/views.py

```diff
@@ -1048,7 +1048,7 @@ def filter_queryset(self, queryset):
         unanswered_by_user = self.request.query_params.get(
             "unanswered_by_user"
         )
-        if unanswered_by_user == "True":
+        if str(unanswered_by_user).lower() == "true":
```

Comment:
`str()` to coerce the type which can be "None" or anything else, I guess, into a string.